media/js/firefox/new/scene1-batm-anim.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
+    'use strict';
+
+    var exp = $('#main-header-copy').data('experience');
+    var mqDesktop;
+    var buttonOpen = '<button type="button" class="open" title="'+ Mozilla.Utils.trans('global-open') +'">'+ Mozilla.Utils.trans('global-open') +'</button>';
+    var buttonNext = '<button type="button" class="next" title="'+ Mozilla.Utils.trans('global-next') +'">'+ Mozilla.Utils.trans('global-next') +'</button>';
+    var allPoints = $('.points .label');
+    var point1 = $('#point-1 .label');
+    var point2 = $('#point-2 .label');
+    var point3 = $('#point-3 .label');
+
+    $('.download-link').each(function(i, link) {
+        if (exp && link.href.indexOf('scene=2') > -1) {
+            // specify v=1 template for scene 2
+            link.href = link.href.replace('scene=2', 'scene=2&xv=' + exp);
+        }
+    });
+
+    // trigger fade-in CSS animation
+    $(function() {
+        $('html').addClass('ready');
+    });
+
+    // Set desktop media query
+    if (typeof matchMedia !== 'undefined') {
+        mqDesktop = matchMedia('(min-width: 1000px)');
+    }
+
+    // Fire the points for desktop and up
+    if (mqDesktop.matches) {
+        setupPoints();
+    }
+
+    // Set or unset points on resize
+    mqDesktop.addListener(function(mq) {
+        if (mq.matches) {
+            setupPoints();
+        } else {
+            unsetPoints();
+        }
+    });
+
+    // Set up three points, two with buttons
+    function setupPoints() {
+        $(buttonOpen).insertBefore([point1, point2]); // Add the open buttons
+        $(buttonNext).appendTo([point1, point2]); // Add the next buttons
+
+        var point1Next = point1.find('.next');
+        var point2Next = point2.find('.next');
+
+        // Show the first point
+        point1.show();
+
+        // Hide point 1, show point 2
+        point1Next.on('click', function() {
+            point1.fadeOut(100);
+            point2.fadeIn(100);
```

Comment:
Not a blocker, but it would be nice to move focus to the heading of the next popup when it opens?